geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -176,8 +176,18 @@
   private static final int prMetaDataSentCountId;
 
   private static final int localMaxMemoryId;
+  static final int bucketClearsId;
+  static final int bucketClearTimeId;
+  static final int bucketClearsInProgressId;
 
   static {
+    final String bucketClearsDesc =
+        "The total number of times a clear has been done on this region and it's bucket regions";
+    final String bucketClearTimeDesc =
+        "The total amount of time spent doing a bucket clear onx bucket regions";
+    final String bucketClearsInProgressDesc =
+        "The total number bucket region clears in progress";
```

Comment:
Counters are "totals". Gauges are "current". So: "The current number bucket clears on this partitioned region that are in progress."
We want to avoid the term "bucket region" in a descriptions. We have a partitioned region that has buckets

Reply:
Good feedback. Thanks!